### PR TITLE
Try three times to get a tweet before raising an error.

### PR DESCRIPTION
**Pull request by progval**

### CODE REVIEW

File: backend/realtime_worker.py [hunk 1]
```diff
@@ -23,7 +26,8 @@ def fetch_tweets(db, stream):
         print ('Got tweet.')
         if isTransportOffer(tweet):
             print ('Adding tweet from @{}'.format(tweet['user']['screen_name']))
-            r.db('voyageavecmoi').table('offers').insert(get_tweet_data(tweet)).run(db)
+            tweet_data = retry_call(get_tweet_data, (tweet,), tries=MAX_TRIES)
```
Comment (RaitoBezarius) [inline backend/realtime_worker.py]: Why do you retry to extract tweet data if you already have the data?
The `get_tweet_data` function is somehow side-effect free, right?

Comment (progval) [inline backend/realtime_worker.py]: Hmm, good point, the issue was with `stream.statuses.filter(track='#VoyageAvecMoi')` actually (but I could not reproduce the error from Twitter so I couldn't test it)

Comment (RaitoBezarius) [inline backend/realtime_worker.py]: Alright, I think that we should wrap the iteration in a `try/catch` and make somehow restart the loop.

**Comment by progval** [comment]
Fixed the issue you pointed out by retrying the actual failing line (the for loop)

**Comment by progval** [comment]
Note that now it will try indefinitely, with an exponential backoff

**Comment by progval** [comment]
Actually, I am not sure this is the right way, because this does not report success to the library, so even after a success, the backoff will be increased.
Any better idea?

**Comment by RaitoBezarius** [comment]
I do not know exactly how `retry` works, but basically whenever an exception is thrown, retry catch it and recall the function, right?

**Comment by RaitoBezarius** [comment]
I think that for now, it should do it.

**Comment by progval** [comment]
Yes. But let me remove the backoff increase first

**Comment by RaitoBezarius** [comment]
Alright, let me know when I can merge the PR.

**Comment by progval** [comment]
Done.

**Comment by RaitoBezarius** [comment]
Thanks !

**Pull request merged**